.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "python.linting.enabled": true,
+    "python.testing.pytestArgs": [
+        "tests"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
```

Comment:
but we use unittest, why is this referencing pytest?

Reply:
Done